Sources/VSM/StateContainer/StateContainer.swift

```diff
@@ -68,7 +68,8 @@ final public class StateContainer<State>: ObservableObject, StateContaining {
 // MARK: - Observe Function Overloads
 
 public extension StateContainer {
-    /// Observes the state publisher emitted as a result of invoking some action
+    
+    // See StateObserving for details
```

Comment:
You should provide a DocC link to `StateObserving` from here. Improves the ergonomics of using the popup help. (Applies to a bunch of places below here)